modules/custom/social_ajax_comments/src/Controller/AjaxCommentsController.php

```diff
@@ -181,6 +181,13 @@ public function socialAdd(Request $request, EntityInterface $entity, $field_name
       // AjaxCommentsForm::save() during the previous HTTP request.
       $cid = $this->tempStore->getCid();
 
+      // Reset cache of the commented entity.
```

Comment:
Hi, @ksis 

Since in the d.org [issue](https://www.drupal.org/project/social/issues/3275943#comment-14485864) you commented that it is a temporary solution could you add @todo, please? 
And also add more description of the problem to the commit message.

Thanks!

Reply:
Hi @Ressinel,

I've updated the PR. Please, check if it would be enough. 

Thanks!